Nightly builds are broken

**Description**

Nigthly builds are broken since mar17, 
https://github.com/containerd/containerd/actions/workflows/nightly.yml

The errors seems to be a dependency on btrfs

```
# github.com/containerd/btrfs
vendor/github.com/containerd/btrfs/btrfs.go:21:10: fatal error: btrfs/ioctl.h: No such file or directory
   21 | #include <btrfs/ioctl.h>
      |    
```

--- a/.github/workflows/nightly.yml
+++ b/.github/workflows/nightly.yml
@@ -52,7 +52,11 @@ jobs:
             libseccomp-dev:amd64 \
             libseccomp-dev:arm64 \
             libseccomp-dev:s390x \
-            libseccomp-dev:ppc64el
+            libseccomp-dev:ppc64el \
+            libbtrfs-dev:amd64 \
+            libbtrfs-dev:arm64 \
+            libbtrfs-dev:s390x \
+            libbtrfs-dev:ppc64el
 
       - name: Build amd64
         env:

